src/pages/add-product/components/field-description/styles.ts

```diff
@@ -0,0 +1,21 @@
+import styled from "@emotion/styled";
+import { Box, Typography } from '@mui/material';
+
+import theme from '~/theme';
+
+const StyledDescriptionBox = styled(Box)`
+    width: 15%;
+`;
+
+const TitleTypography = styled(Typography)`
+  color: ${theme.palette.primary.main};
+  margin-bottom: 8px;
+  font-size: ${theme.typography.playfairDisplayBold};
+`;
+
+const DescriptionTypography = styled(Typography)`
+  color: ${theme.palette.fontGray};
+  font-size: ${theme.typography.dmSans};
+`;
```

Comment:
For all styles with theme.
```suggestion
const DescriptionTypography = styled(Typography)(({ theme }) => ({
  color: theme.palette.fontGray,
  fontSize: theme.typography.dmSans,
}));
```
or 
```suggestion
const DescriptionTypography = styled(Typography)(({ theme }) => ({
  color: theme.palette.fontGray,
  ...theme.typography.dmSans,
}));
```